### PR TITLE
Fix up Typescript SDK examples

### PR DESCRIPTION
**Pull request by jkaster**
The move to the monorepo caused some run-time issues for me. This resolves them.

### CODE REVIEW

**Review by bryans99 — APPROVED** [review]
LGTM but I have not tested them out. Might want to wait for another review

**Review by josephaxisa — APPROVED** [review]
LGTM. Also gave this a try and it worked like a charm.

**Pull request merged**
